dependencies.props

```diff
@@ -37,8 +37,8 @@
     <ProjectNTfsExpectedPrerelease>beta-26007-00</ProjectNTfsExpectedPrerelease>
     <ProjectNTfsTestILCExpectedPrerelease>beta-26007-00</ProjectNTfsTestILCExpectedPrerelease>
     <ProjectNTfsTestILCPackageVersion>1.0.0-beta-26007-00</ProjectNTfsTestILCPackageVersion>
-    <MicrosoftNETCoreDotNetHostPackageVersion>2.0.1-servicing-25615-03</MicrosoftNETCoreDotNetHostPackageVersion>
-    <MicrosoftNETCoreDotNetHostPolicyPackageVersion>2.0.1-servicing-25615-03</MicrosoftNETCoreDotNetHostPolicyPackageVersion>
+    <MicrosoftNETCoreDotNetHostPackageVersion>2.1.0-preview1-26007-02</MicrosoftNETCoreDotNetHostPackageVersion>
+    <MicrosoftNETCoreDotNetHostPolicyPackageVersion>2.1.0-preview1-26007-02</MicrosoftNETCoreDotNetHostPolicyPackageVersion>
     <MicrosoftNETCoreAppPackageVersion>2.1.0-preview1-26007-02</MicrosoftNETCoreAppPackageVersion>
```

Comment:
How is the NETCore app package version updated?  Is it manual too?

If these are all manually updated, maybe we should have the version in one place, and then the other properties read it from the one place.

That way they don’t get out of sync again. But can diverge if needed in the future

Reply:
Something like 

MicrosoftNETCoreAppPackageVersion=2.1.0-preview1...
MicrosoftNETCoreDotNetHostPackageVersion=$(MicrosoftNETCoreAppPackageVersion)
MicrosoftNETCoreDotNetHostPolicyPackageVersion=$(MicrosoftNETCoreAppPackageVersion)

Reply:
It looks like @dotnet-maestro-bot updates the MicrosoftNETCoreAppPackageVersion

Reply:
@eerhardt the change in the versions I've made causes all the Linux and OSX tests fail in Invariant.Tests tests.
Also, on Alpine, while it fixed the problem with restoring the dotnet host stuff and the job successfully compiled native code in the build phase, it then fails with the following error (both in the CI and locally on my Linux box:
```
/home/janvorli/git/corefx/src/dirs.proj(35,5): error : Unable to find platform package that matches runtime.alpine.3.6-x64.Microsoft.NETCore.App when generating deps.json file.
/home/janvorli/git/corefx/src/dirs.proj(35,5): error MSB4018: The "GenerateDepsJson" task failed unexpectedly.
/home/janvorli/git/corefx/src/dirs.proj(35,5): error MSB4018: System.NullReferenceException: Object reference not set to an instance of an object.
/home/janvorli/git/corefx/src/dirs.proj(35,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.GenerateDepsJson.Execute()
/home/janvorli/git/corefx/src/dirs.proj(35,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
/home/janvorli/git/corefx/src/dirs.proj(35,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__25.MoveNext()

Build FAILED.
```
I'm not sure what's going on here. I wonder if it is related to the fact that if I look into the packages folder for `microsoft.netcore.app`, I can see there is only version 2.0.0 present and that one knows nothing about Alpine.

Reply:
Microsoft.NETCore.App is updated via maestro. https://github.com/janvorli/corefx/blob/db9563b8e7a94cb890a2a9da7c500e5833665b1b/dependencies.props#L151 we should add the host packages to get updates as well. 

>I'm not sure what's going on here. I wonder if it is related to the fact that if I look into the packages folder for microsoft.netcore.app, I can see there is only version 2.0.0 present and that one knows nothing about Alpine.

I suspect that might be part of it as we try and setup the test deps file based on the cli we are running on https://github.com/janvorli/corefx/blob/master/src/src.builds#L15. I don't know why that wasn't failing before though. When you updated the alpine CLI did not include runtime.alpine.3.6-x64.Microsoft.NETCore.App in the shared framework?

Reply:
@weshaggard I am not sure if I understand the question, but the Alpine bootstrap CLI contains shared/Microsoft.NETCore.App/2.0.0/Microsoft.NETCore.App.deps.json that does contain alpine.3.6-x64 in the runtimes:  section like this:
```json
    "alpine.3.6-x64": [
      "unix",
      "unix-x64",
      "any",
      "base"
    ]
```

Reply:
Hmm, it actually looks like it is a problem of the bootstrap cli. Not due to the runtimes section, but due to the "runtime" section where the linux-x64 was not changed to alpine.3.6. That was due to the old version of my bootstrapping script and it used to work fine in the past, so probably there is some new mechanism that now fails. 
I've tried to modify the Microsoft.NETCore.App.deps.json in Tools/dotnetcli/shared/.... folder locally and it fixed the build issue.
So I'll patch the bootstrap CLI and retry in an hour after the Azure cache expires.